azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/service/impl/AbstractAppService.java

```diff
@@ -31,75 +30,76 @@
 import java.io.InputStream;
 import java.nio.ByteBuffer;
 import java.util.List;
-import java.util.Objects;
 import java.util.Optional;
 
-abstract class AbstractAppService<T extends WebAppBase, R extends AppServiceBaseEntity> implements IAppService<R> {
+abstract class AbstractAppService<T extends WebAppBase, R extends AppServiceBaseEntity> extends AbstractAzureManager<T> implements IAppService<R> {
 
+    protected static final String APP_SERVICE_ID_TEMPLATE = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Web/sites/%s";
     protected AppServiceKuduManager kuduManager;
     protected R entity;
     protected T remote;
```

Comment:
is it possible to move these 2 fields to parent class?

Reply:
Keep entity here as we plan to remove it later, moved remote to parent